CHANGELOG.md

```diff
@@ -35,8 +35,9 @@
 ### Improvements
 
 - **Azure Queue:** Don't call Azure queue GetProperties API unnecessarily ([#2613](https://github.com/kedacore/keda/pull/2613))
-- **Kafka Scaler** Make "disable" a valid value for tls auth parameter ([#2608](https://github.com/kedacore/keda/issues/2608))
 - **Datadog Scaler:** Validate query to contain `{` to prevent panic on invalid query ([#2625](https://github.com/kedacore/keda/issues/2625))
+- **Kafka Scaler** Make "disable" a valid value for tls auth parameter ([#2608](https://github.com/kedacore/keda/issues/2608))
+- **Memory Scaler** Adding e2e test for the memory scaler ([#2220](https://github.com/kedacore/keda/issues/2220))
```

Comment:
Please add this to ### Other section as it is nod directly user facing, thanks!

Reply:
Hi @zroubalik , do you mean only memory scaler one or both kafka and memory scaler one too?

Reply:
@zroubalik Hi, I'll add the memory scaler entry to Other section.

Reply:
yeah, just Memory scaler

Reply:
Done in the latest update
@zroubalik